### PR TITLE
Add KB: After upgrading Longhorn, Volume Cannot Attach Because of The Leftover non-empty volume.status.PendingNodeID or Longhorn UI Attachment Ticket

### PR DESCRIPTION
**Pull request by PhanLe1010**
longhorn/longhorn#7994
longhorn/longhorn#8339

### CODE REVIEW

File: content/kb/troubleshooting-volume-cannot-attach-due-to-orphan-pending-node-id-or-longhorn-ui-ad-ticket.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+## Solution
+
+### Longhorn volume cannot attach because of the leftover non-empty `volume.status.PendingNodeID`
+
+We fixed this bug in Longhorn version >= v1.5.5. If you hit this bug in v1.5.0/v1.5.1/v1.5.2/v1.5.3/v1.5.4, you can use the workaround: run `kubectl edit volumes -n longhorn-system VOLUME-NAME --subresource=status` and set `volume.status.PendingNodeID` to empty string `""`
+
+### Longhorn volume cannot attach to new node due to unexpected longhorn-ui attachment ticket
+
+The root cause of this issue is still being investigated at https://github.com/longhorn/longhorn/issues/8339. If you hit this issue, you can use this workaround:
+1. `kubectl edit volumes -n longhorn-system VOLUME-NAME`
```
Comment (innobead) [inline content/kb/troubleshooting-volume-cannot-attach-due-to-orphan-pending-node-id-or-longhorn-ui-ad-ticket.md:38]: Is this unnecessary? as users can just check the volume attachment and update it.

**Review by innobead — COMMENTED** [review]
LGTM, just one question.

File: content/kb/troubleshooting-volume-cannot-attach-due-to-orphan-pending-node-id-or-longhorn-ui-ad-ticket.md (hunk 1 above)
Comment (PhanLe1010) [reply content/kb/troubleshooting-volume-cannot-attach-due-to-orphan-pending-node-id-or-longhorn-ui-ad-ticket.md]: OOps. You are right. My mistake. Fixed. Thank you

**Review by innobead — DISMISSED** [review]
LGTM. Waiting for @jillian-maroket review.

File: content/kb/troubleshooting-volume-cannot-attach-due-to-orphan-pending-node-id-or-longhorn-ui-ad-ticket.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+title: "After upgrading Longhorn, Volume Cannot Attach Because of The Leftover non-empty volume.status.PendingNodeID or Longhorn UI Attachment Ticket"
```
[the remark this answers is not in the document]
Comment (jillian-maroket) [reply content/kb/troubleshooting-volume-cannot-attach-due-to-orphan-pending-node-id-or-longhorn-ui-ad-ticket.md:2]: The original title is too long. Describe the issue clearly and then save the other information for the article.

**Review by jillian-maroket — COMMENTED** [review]
I restructured some parts because of repetitive information.

**Review by innobead — APPROVED** [review]

**Pull request merged**